cmd/build-extensions-container.go

```diff
@@ -185,8 +185,32 @@ func buildExtensionContainer() error {
 		SizeInBytes:     float64(stat.Size()),
 		SkipCompression: true,
 	}
-	cosaBuild.MetaStamp = float64(time.Now().UnixNano())
 
+	fmt.Printf("Generating meta.json `extensions entry for`: %s\n", buildID)
+	extensionsFilePath:= "./tmp/extensions.json"
```

Comment:
mention here in a comment when/where this file is created (i.e. link to the Dockerfile)